contracts/interfaces/IDCAHubCompanion.sol

```diff
@@ -192,10 +221,7 @@ interface IDCAHubCompanionHubProxyHandler {
   ) external payable;
 }
 
-interface IDCAHubCompanion is IDCAHubCompanionLibrariesHandler, IDCAHubCompanionHubProxyHandler {
-  /// @notice Thrown when one of the parameters is a zero address
-  error ZeroAddress();
-
+interface IDCAHubCompanion is IDCAHubCompanionLibrariesHandler, IDCAHubCompanionHubProxyHandler, IDCAHubCompanionTakeWithdrawAndSwapHandler {
   /// @notice Thrown when a user tries operate on a position that they don't have access to
   error UnauthorizedCaller();
```

Comment:
Does this need to be here? Or could we move this to `IDCAHubCompanionHubProxyHandler`?